.github/ISSUE_TEMPLATE/workpro_bug.md

```diff
@@ -0,0 +1,45 @@
+---
+name: WorkPro-bug-report
+about: wechaty-puppet-workpro bug report
+title: '[Bug]WorkPro:'
```

Comment:
To make sure this is consistent, we should use `BUG` instead of `Bug`, since the feature template is using `FEAT`.